README.md

```diff
@@ -52,7 +52,7 @@ previous stable release.
 ### Foreman version compatibility notes
 
 This module targets Foreman 1.21+. Running without passenger is only supported
-on Foreman 1.22+.
+on Foreman 1.23+.
```

Comment:
We didn't cherry pick https://github.com/theforeman/foreman-packaging/commit/ab3a75c954f616944199a68a2a76b86acd15aa6f into 1.22. So for RPMs it's 1.22 but debs 1.23. Rather than cherry picking it for a less used function I'm choosing to just mark it as 1.23.